install/certmonger/dogtag-ipa-ca-renew-agent-submit

```diff
@@ -65,8 +65,38 @@ if six.PY3:
 IPA_CA_NICKNAME = 'caSigningCert cert-pki-ca'
 
 def get_nickname():
-    csr = os.environ.get('CERTMONGER_CSR')
-    return pkcs10.get_friendlyname(csr) if csr else None
+    subject = os.environ.get('CERTMONGER_REQ_SUBJECT')
+    if not subject:
+        return None
+
+    subject_base = dsinstance.DsInstance().find_subject_base()
+    if not subject_base:
+        return None
+
+    nickname_by_subject_dn = {
```

Comment:
By DN I did not mean the subject DN of the certificate, but the DN of the LDAP entry - it must remain static, otherwise renewal wouldn't work correctly.